staging/src/k8s.io/kube-apiextensions-server/pkg/controller/status/naming_controller.go

```diff
@@ -214,6 +214,30 @@ func (c *NamingConditionController) sync(key string) error {
 		return err
 	}
 
+	crd := &apiextensions.CustomResourceDefinition{}
+	if err := apiextensions.DeepCopy_apiextensions_CustomResourceDefinition(inCustomResourceDefinition, crd, cloner); err != nil {
+		return err
+	}
+
+	// when a resource is deleted, enqueue other resources of the same group
+	for _, condition := range crd.Status.Conditions {
+		if condition.Type == "Terminating" && condition.Status == "False" {
+			// wait for all accepted names to be updated
+			time.Sleep(1 * time.Second)
+			list, err := c.crdLister.List(labels.Everything())
+			if err != nil {
+				return nil
+			}
+			for _, curr := range list {
```

Comment:
can we turn this for loop into a `RequeueAllOtherGroupCRDs`? We can then reuse it at the bottom of the func.